asistente_ladm_col/utils/decorators.py

```diff
@@ -166,6 +166,19 @@ def decorated_function(*args, **kwargs):
 
     return decorated_function
 
+def _grass7_required(func_to_decorate):
+    @wraps(func_to_decorate)
+    def decorated_function(*args, **kwargs):
+        try:
+            import processing
+            grass7 = processing.algs.grass7.Grass7Utils.Grass7Utils
+            print(grass7.installedVersion())
```

Comment:
Remove print.

Reply:
Thanks for the adjustment, This was updated in the commit 7830fec